chart/templates/_helpers.yaml

```diff
@@ -357,6 +357,10 @@ server_tls_key_file = /etc/pgbouncer/server.key
 {{ (printf "%s/config/airflow_local_settings.py" .Values.airflowHome) | quote }}
 {{- end }}
 
+{{ define "airflow_install_requirements_path" -}}
```

Comment:
The name should be change to match the naming we already have.

In our Dockerfile we already have ADDITIONAL_PYTHON_DEPS variable and we are installing additional python dependencies. @mik-laj rightfully noticed that not alll deps can be installed this way (we do not have build-essentials in the image for one) but some can be, and I agree it's nice to have, them, it should be clear however, that those are additional dependencies, and they are not de-facto "install_requirements". The "install_requirements" in python world have a very precise meaning - those are to be found in `setup.py` or `setup.cfg` and they mean "those need to be installed BEFORE you install airflow". What we are talking about here are "additional python dependencies" which are installed AFTER airflow is installed. So let's name it this way.

Reply:
This is of course slightly different than the Dockerfile ones (those are embedded in the customized image) but the meaning is the same, so name should also be the same.